templates/_layouts/docs.html

```diff
@@ -43,6 +43,7 @@
               {{ side_nav_item("/docs/building-vanilla", "Building with Vanilla") }}
               {{ side_nav_item("/docs/customising-vanilla", "Customising Vanilla") }}
               {{ side_nav_item("/docs/whats-new", "What’s new in " ~ version) }}
+              {{ side_nav_item("/docs/upgrade-guide-v3", "Vanilla 3.0 upgrade guide") }}
```

Comment:
We should make it prominent:

```suggestion
              {{ side_nav_item("/docs/upgrade-guide-v3", "Vanilla 3.0 upgrade guide", "new") }}
```